pkg/simple/client/devops/jenkins/options.go

```diff
@@ -31,8 +31,7 @@ type Options struct {
 	MaxConnections             int    `json:"maxConnections,omitempty" yaml:"maxConnections" description:"Maximum connections allowed to connect to Jenkins"`
 	DevOpsServiceAddress       string `json:"devopsServiceAddress" yaml:"devopsServiceAddress" description:"The separate DevOps Service address"`
 	DevOpsPluginServiceAddress string `json:"devopsPluginServiceAddress" yaml:"devopsPluginServiceAddress" description:"The KubeSphere DevOps plugin Service address"`
-	// K8sBearerToken does not take input from outside
```

Comment:
Please don't forget my comment. This field does not accept the outside input. So, it's not necessary to add the json tag.

Reply:
I misunderstood your meaning.